### DISCUSSION TITLE
Add Save Button in List View (to update the text field value in DB)

### DISCUSSION THREAD

**Discussion by sandeepchowdary7** [Laravel-Backpack/community-forum#308] [Q&A (Help)]
Hi, I was trying to add the Save button in ListView. So that I can enter the value in the text field and click on Save Button and save it in DB. this is what I'm trying to achieve but it is not working as expected.

**Controller**
`$this->crud->addButtonFromModelFunction('line', 'Save', 'showSaveButton', 'beginning');`

**Model**
`public function showSaveButton()
    {

        return '<a href="'.backpack_url('/student/'.$this->id).'" title="Save"><i class="fas fa-desktop"></i> Save </a>';
    }`

**Route**
`Route::post('/student/{id}', 'StudentCrudController@update');`

**Controller-Update Method**
`public function update()
    {

        dd($this);

        return $this->traitUpdate();
    }`

_but getting the result is empty. how I can achieve this. any help or suggestions would be great. Thanks!_

**Comment by pxpm** [comment]
Hello @sandeepchowdary7 

What result is empty ? `dd($this)` ? 

Controller is not called ? Where is the field in list ? 

Pedro

**Comment by tabacitu** [comment]
@sandeepchowdary7 this doesn't look like a bug to me, more of an implementation issue, so I'll move it to discussions. That being said, there aren't that many people in Discussions so I'd recommend asking this kind of questions on StackOverflow - much more people there.

Cheers!

**Comment by support** [comment]
Oups!

Looks like this is a support request, not a bug/feature. **Could you please [repost on StackOverflow](https://stackoverflow.com/questions/tagged/backpack-for-laravel), using the ```backpack-for-laravel``` tag**?

Background: **Here at Backpack we use Github Issues only for tracking bugs and features**, not individual implementation issues. This helps _a lot_ in keeping our focus on improving Backpack. Thanks a lot for understanding!

Here are all the Backpack communication mediums:
 - Long questions (_I have done X and Y and it won't do Z wtf_) - [Stackoverflow](https://stackoverflow.com/questions/tagged/backpack-for-laravel), using the ```backpack-for-laravel``` tag; this is recommended for most questions, since other developers can then find the answer on a simple Google search; also, people get points for answering - and who doesn't like StackOverflow points?!
 - Quick help (_How do I do X_) - [Gitter Chatroom](https://gitter.im/BackpackForLaravel/Lobby);
 - Bug Reports, Feature Requests - Github Issues (here);

Please keep in mind Backpack offers no official / paid support. Whatever help you receive here, on Gitter, Slack or StackOverflow is thanks to our awesome _awesome_ community members, who give up some of their time to help their peers. If you want to join our community, just start pitching in. We take pride in being a welcoming bunch.

Thank you!

--
Justin Case
The Backpack Robot

PS. In case I mistakenly closed your issue, yell :-) I'm a robot, I make mistakes.

**Comment by jcastroa87** [comment]
Hi.

Due to not activity on many years i will close this discussion, if there is needed feel free to reopen or create a new one.

Cheers.
